api/envoy/config/retry/omit_hosts/v2/omit_hosts_config.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package envoy.config.retry.omit_hosts.v2;
+
+import "envoy/api/v2/core/base.proto";
+
+option java_package = "io.envoyproxy.envoy.config.retry.omit_hosts.v2";
+option java_outer_classname = "OmitHostsConfigProto";
+option java_multiple_files = true;
+
+// [#protodoc-title: Omit hosts retry predicate]
+
+// A retry host predicate that can be used to reject a host based on
+// predefined metadata match criteria
+// [#extension: envoy.retry_host_predicates.omit_hosts]
+message OmitHostsConfig {
```

Comment:
I would  call this something like "HostMetadataPredicate" instead of OmitHosts, as all the predicates are mechanisms for omitting certain hosts.

Reply:
Just to clarify, do you mean the extension to be something like: envoy.retry_host_predicates.host_metadata_predicate?

Reply:
yeah or perhaps `envoy.retry_host_predicates.omit_host_metadata` to be more in line with the existing canary host one

Reply:
Ack.